packages/accordions/src/elements/stepper/components/Label.tsx

```diff
@@ -12,7 +12,9 @@ import { StyledLabel, StyledLabelText, StyledIcon, StyledIconFlexContainer } fro
 import { useStepContext, useStepperContext } from '../../../utils';
 
 interface ILabel {
+  /** Replaces the Stepper number with an icon */
```

Comment:
@KICKdesigns my bad. We've determined to use lowercase component names in these descriptions.

> stepper

Reply:
Fixed in latest push